great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -35,6 +36,7 @@
     from great_expectations.render.renderer_configuration import AddParamArgs
 
 
+@public_api
```

Comment:
I think this change unblocks some of @Rachel-Reverie's work, so might want to give her a heads up when this merges.